src/Components/Profile/Header/Header.jsx

```diff
@@ -14,7 +14,7 @@ class Header extends React.Component {
                         <div className='profile-name'>
                             {this.props.profileData.name}
                         </div>
-                        <PlayerStats profileData={this.props.profileData} />
+                        {/* <PlayerStats profileData={this.props.profileData} /> */}
```

Comment:
Just dropping by here since normally I tell you to remove commented out code, but this time I approve :+1: